tests/cache/models/subjects/post.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/cache/models/subjects/post.js
+ */
+'use strict'; // eslint-disable-line strict
+
+const supertest = require('supertest');
+const api = supertest(require('../../../../index').app);
+const path = '/v1/subjects';
+const constants = require('../../../../api/v1/constants');
+const tu = require('../../../testUtils');
+const rtu = require('../redisTestUtil');
+const samstoinit = rtu.samstoinit;
+const redisClient = rtu.redisClient;
+const Subject = tu.db.Subject;
+const expect = require('chai').expect;
+
+describe('redis: subject: POST: ', () => {
```

Comment:
@iamigo @annyhe When tests fail, it is kind of difficult to find out which file the test is in. Any thoughts on adding the file location in the description instead of this. ?